packages/commands-logging/src/write_log.ts

```diff
@@ -0,0 +1,65 @@
+import { Command } from '@seagull/commands'
+import { getCurrentWorkingDirectoryFolder as cwdf } from '@seagull/libraries'
+import { CWLMockFS } from '@seagull/mock-cloudwatchlogs'
+import * as AWS from 'aws-sdk'
+import {
+  PutLogEventsRequest,
+  PutLogEventsResponse,
+  Timestamp,
+} from 'aws-sdk/clients/cloudwatchlogs'
+import { PromiseResult } from 'aws-sdk/lib/request'
+import * as _ from 'lodash'
+import * as moment from 'moment'
+import { CWLSandbox } from './logging_sandbox'
+
+interface Log {
+  message: string | object | number
+  timestamp?: Timestamp
+}
+
+interface WriteLogRequest {
+  logStreamName: string
+  logs: Log[]
+}
+
+/**
+ * Command to write log object to cloudwatch
+ */
+export class WriteLog extends Command<
+  PromiseResult<PutLogEventsResponse, AWS.AWSError>
+> {
+  params: PutLogEventsRequest
+  CWL = new AWS.CloudWatchLogs({ region: process.env.AWS_REGION })
```

Comment:
you're the first to use aws region
not sure if i like process.env, it should be there on a lambda; it is not on edge but should be no problem for now (will be initialized anyway)

Reply:
you can't initialize it without region. But you are right, it is undefined on edge. should i hardcode it for now and solve it with the config package later?